concrete/attributes/topics/controller.php

```diff
@@ -134,9 +143,12 @@ public function setNodes($akTopicParentNodeID, $akTopicTreeID)
 
     public function createAttributeValue($nodes)
     {
-        $selected = array();
+        $selected = [];
         $this->load();
         $tree = Tree::getByID($this->akTopicTreeID);
+        if (is_array($nodes) && $this->akTopicAllowMultipleValues == false) {
+            $nodes = $nodes[0];
```

Comment:
I think developers can user `createAttributeValue` method for their needs. `createAttributeValueFromRequest` for submit forms. If you say it is not necessary to add `is_array`, I'll remove it ;-)